unittests/test_exec_finder.py

```diff
@@ -13,7 +13,11 @@ def test_create_and_config(tmpdir):
 
 def test_find_in_no_places():
     ef = ExecutableFinder([])
-    assert ef.get_executable_path("abc.aplx") is None
+    try:
+        ef.get_executable_path("abc.aplx")
+        return False
+    except Exception:
+        return True
```

Comment:
This is not how you assert that an exception happened!

Reply:
fixed